### DISCUSSION TITLE
RecieverInfo는 Address와 관련이 있는것 같습니다.

### DISCUSSION THREAD

**Discussion by wenodev** [CodeSoom/DDD-Kurly-Clone-Order#16] [Ideas]
- 배송지 정보에 받는사람 정보가 들어가는게 좋다고 생각합니다.
![image](https://user-images.githubusercontent.com/35131529/139004818-2cec574f-1652-4442-beee-b6d540992483.png)

**Comment by joowankim** [comment]
`DeliveryInfo`라는 배송지 정보에 이미 `ReceiverInfo`가 `DeliveryDetails`에 포함되어 들어가있는 상태입니다. 어떻게 들어가 있는게 좋은것인지 좀 더 자세히 말씀해주실 수 있을까요?
